styles/style.css

```diff
@@ -0,0 +1,687 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  font-family: 'montserrat', sans-serif;
+}
+
+a {
+  color: #000;
+  text-decoration: none;
+}
+
+.wrapper {
+  width: 100%;
+  height: 100%;
+  min-width: 370px;
+  max-width: 1440px;
+  margin: 0 auto;
+}
+
+.header {
+  margin-bottom: 31px;
+  white-space: nowrap;
+}
+
+.nav {
+  display: flex;
+  justify-content: space-between;
+  padding: 26px 40px 0 46px;
+  margin-bottom: 42px;
+}
+
+.logo {
+  margin-right: 2vw;
+}
+
+.header-models {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  font-size: 15px;
+  font-weight: 600;
+  font-style: normal;
+  white-space: nowrap;
+}
+
+.model {
+  margin: 0 24px 3px 18px;
+  padding-bottom: 10px;
+}
+
+.model-s__link:first-child {
+  margin-right: 5px;
+}
+
+.model:hover {
+  color: #e51937;
+  border-bottom: 2px solid;
+  padding: 0 0 6px 0;
+}
+
+.model-s__link {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected selector ".model-s__link" to come before selector ".model-s__link:first-child" (no-descending-specificity)](https://app.codacy.com/gh/WebHeroSchool/Tesla_Privezentsev/pullRequest?prid=5491648)